.github/workflows/reusable-build-test-release.yml

```diff
@@ -385,11 +299,9 @@ jobs:
   semgrep:
     runs-on: ubuntu-latest
     name: security-sast-semgrep
-    needs:
-      - setup-workflow
+    if: github.actor != 'dependabot[bot]'
```

Comment:
Why this condition?

Reply:
> Why this condition?

@kdoroszko-splunk it is to ensures that the job named security-sast-semgrep is only executed when the actor triggering the workflow is not the Dependabot GitHub bot. if that will create an shall i remove that?

Reply:
AFAIK we don't use dependabot? Do we want to keep this? @kdoroszko-splunk @artemrys

Reply:
I removed this on the beginning of January in this [PR](https://github.com/splunk/addonfactory-workflow-addon-release/pull/213). So this is why I asked.

Reply:
@mbruzda-splunk please remove this condition

Reply:
removed